File: demo/md/md.cpp
#include "mpi.h"
#include "CPL.h"

int main (int argc, char** argv)
{
    MPI_Init (&argc, &argv);
    
    MPI_Comm realmComm;
    CPL::create_comm (CPL::md_realm, realmComm);

    MPI_Finalize();
}

File: demo/md/md.f90
program main
    use mpi
    use cpl, only : md_realm, CPL_create_comm
    implicit none

    integer :: rank, ierror, MD_COMM

    call MPI_Init(ierror)
    call CPL_create_comm(md_realm, MD_COMM, ierror)
    call MPI_Finalize(ierror)

end program main 
